packages/renderer/src/lib/pod/PodsList.spec.ts

```diff
@@ -22,7 +22,10 @@ import '@testing-library/jest-dom/vitest';
 
 import { fireEvent, render, screen } from '@testing-library/svelte';
 import userEvent from '@testing-library/user-event';
+/* eslint-disable import/no-duplicates */
```

Comment:
no duplicates ?

Reply:
I commented before it's for svelte,  svelte/store imports